iati_datastore/iatilib/parse.py

```diff
@@ -401,25 +399,6 @@ def default_language(xml, resource=None, major_version='1'):
     return codelists.by_major_version[major_version].Language.from_string(xml_value)
 
 
-def _open_resource_from_file(filename):
```

Comment:
Are we sure this function isn't used elsewhere? I guess not though if tests are passing!

Reply:
Yep, this is safe to remove.